### ISSUE TITLE
wrenfmt

### ISSUE THREAD

**Issue by kmarekspartz** [wren-lang/wren#89]
After #87 and #88, it would be nice to have a wrenfmt program which would standardize syntax/indentation. This could also allow for migration to newer syntaxes, which could lower the barrier to entry for such a young language.

**Comment by munificent** [comment]
Yes. Strangely enough, I have spent the past three months implementing exactly that for Dart. It's surprisingly difficult.

**Comment by gsmaverick** [comment]
Having used something similar in Scala I'm a big fan of automatic code formatting both for convenience reasons but also for warding off unnecessary formatting holy wars.

**Comment by munificent** [comment]
:+1: to that.

**Comment by underscorediscovery** [comment]
Closing with `future` tag for organizational purposes.
